thirdparty/README.md

```diff
@@ -143,6 +143,15 @@ See the header of glad.c for instructions on how to generate them for
 the GLES version Godot targets.
 
 
+## glslang
+
+- Upstream: https://github.com/KhronosGroup/glslang
+- Version: rev.3226
+- License: glslang
+
+Important: File `glslang/glslang/Include/Common.h` has
+Godot-made change marked with `// -- GODOT --` comments.
+
```

Comment:
Sections are supposed to be separated by two lines (but I should likely drop this convention as literally nobody understands it :P).

Reply:
Fixed by #31102.

Reply:
Fixed by #31102.